src/options/help.rs

```diff
@@ -65,6 +65,7 @@ LONG VIEW OPTIONS
   -U, --created            use the created timestamp field
   --changed                use the changed timestamp field
   --time-style             how to format timestamps (default, iso, long-iso, full-iso, relative, or a custom style with '+' as prefix. Ex: '+%Y/%m/%d')
+  --total-size             show recursive directory size
```

Comment:
show listed directory size is better I think

Reply:
It's show recursive directory size for every other entry in the manpage, etc. idk I think `total-size` plus `recursive` gives a pretty good understanding of what it's doing. Listed directory size might be a little too ambiguous

Reply:
I'm not sure what is meant by a "listed" directory size.

Reply:
Could be something like "shows size of a directory as size of all files inside"? I can't phrase it better thou